### PR TITLE
John/install

### PR DESCRIPTION
**Pull request by Mac-Genius**
* Updates the current Ubuntu setup script
* Adds a MacOS setup script

### CODE REVIEW

File: setup/setup_macos/macos_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# Check if python3 is installed
+if brew ls --versions python3 > /dev/null
+then
+	echo "python3 already installed"
+else
+	echo "Installing python3"
+	brew install python3
+fi
+
+# Check if python3 is installed
```
Comment (BrandonLMorris) [inline setup/setup_macos/macos_dependencies.sh]: This should say `mysql`

File: setup/macos_setup.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+sh setup_macos/macos_dependencies.sh
```
Comment (BrandonLMorris) [inline setup/macos_setup.sh:1]: Any reason why these are explicitly executed with `sh`?

File: setup/macos_setup.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+sh setup_all/initialize_database.sh
+sh setup_all/initialize_config.sh
+sh setup_all/initialize_virtualenv.sh
+# Finished building the website. You can now run ./run.py
```
Comment (BrandonLMorris) [inline setup/macos_setup.sh]: This should probably be echoed

**Review by BrandonLMorris — COMMENTED** [review]
Looks good. A couple minor things

File: setup/macos_setup.sh (hunk 2 above)
Comment (Mac-Genius) [reply setup/macos_setup.sh:1]: I was just copying what was already in the scripts. Do you have a suggested way of executing the other scripts?

Comment (BrandonLMorris) [reply setup/macos_setup.sh:1]: Ah, I see. All good

File: setup/setup_all/initialize_config.sh [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	then
+		echo "Resetting config..."
+		create_config
+	else
+		echo "Not resetting config."
+	fi
+else 
+	echo "Creating config file..."
+	create_config
+fi
```
Comment (mcprice30) [inline setup/setup_all/initialize_config.sh]: Nit - Github says you don't have a newline here.

**Review by mcprice30 — COMMENTED** [review]
Assuming that data.zip includes the default profile picture now, looks good!

File: setup/setup_all/initialize_config.sh (hunk 4 above)
Comment (Mac-Genius) [reply setup/setup_all/initialize_config.sh]: Fixed

**Pull request merged**